wikimedia/wikipedia-ios
# Watchlist Filters PR Feedback

**Phabricator:** 
https://phabricator.wikimedia.org/T342929#9148645
https://phabricator.wikimedia.org/T342929#9150414

### Notes
Various design and bug fixes for Watchlist Filters. See Dependent PR for more notes.

Dependent PRs:
https://github.com/wikimedia/wikipedia-ios-components/pull/28
~~https://github.com/wikimedia/wikipedia-ios-data/pull/11~~ (merged)


### Test Steps
1. Open Watchlist filters when iOS settings are in Dark Mode. Select Light theme. Confirm Filters colors look right. Also test when iOS settings are in Light mode, with app Settings set to Dark or Black theme. Confirm Filters colors look right.
2. Confirm Done button is semibold with the link color.
3. Confirm "All" option is removed from Latest Revisions section.
4. Confirm turning off all projects, and selecting additional filters in other sections correctly updates the filters count on the Watchlist list view.
